src/04-modality-specific-files/10-eye-tracking.md

```diff
@@ -0,0 +1,187 @@
+# Eye-tracking including gaze position and pupil size
+
+Support for eye-tracking dataset developed as a [BIDS Extension Proposal](../07-extensions.md#bids-extension-proposals). Please see [Citing BIDS](../01-introduction.md#citing-bids) on how to appropriately credit this extension when referring to it in the context of the academic literature.
+
+## Terminology and conventions
+
+Eye-tracking-BIDS is fully consistent with the BIDS specification as a whole. However, BIDS was initially developed in the context of MRI, so some terminology may be unfamiliar to researchers from other fields. This section adds clarifications to [Common Principles - Definitions](../02-common-principles.html) for the eye-tracking context.
+
+-  __Eye-tracker__ - refer to the apparatus allowing the records of gaze position and/or pupil size. 
+-	__Screen__ - Video display used to present visual stimulation (for example projector, monitor, tablet).
+
+## Eye-tracking data
+
+```Text
+sub-<label>/
+    [ses-<label>/]
+        <datatype>/
+            sub-<label>[_ses-<label>][_task-<label>][_acq-<label>][_run-<index>]_eyetrack.<datatype>
+            sub-<label>[_ses-<label>][_task-<label>][_acq-<label>][_run-<index>]_eyetrack.json
+            sub-<label>[_ses-<label>][_task-<label>][_acq-<label>][_run-<index>]_events.json
+            sub-<label>[_ses-<label>][_task-<label>][_acq-<label>][_run-<index>]_events.tsv
+```
+
+The eye-tracking data files generally contain gaze position (x/y coordinates) and pupil size data. With Eye-Tracking-BIDS, we wish to promote the adoption of good practices in the management of scientific data. Hence, the current emphasis of Eye-Tracking-BIDS is not to impose a generic data format for the modality, but to standardize the way data is stored in repositories. 
+
+Eye-tracking data MUST be stored in the main data recording modality or `<datatype>` directory (for example func, anat, dwi, meg, eeg, ieeg, or beh). The current version of this extension *does not* include a specification of the extension format and data formatting of recorded eye-tracking data. Thus, data must be stored in an open format (for example EDF file converted in ASCII `.asc` format or `.tsv` etc) with the `_eyetrack` suffix.
+
+We encourage every user to put the raw data before conversion within the `/sourcedata` folder (for example put property EDF files in the sourcedata folder).
```

Comment:
proprietary? please ignore otherwise ignore.

Also, EDF is a valid format for EEG data, therefore one could think that could also be valid here. I would then reinforce my previous comment on which data formats are allowed for this extension

```suggestion
We encourage every user to put the raw data before conversion within the `/sourcedata` folder (for example put proprietary EDF files in the sourcedata folder).
```

Reply:
EDF here is for Eyelink Data Format. Thanks for the correction.
I'm not sure that this is the same in EEG (?).
The rationale of this BEP is that we want people to organize their data following BIDS standards but not to reformat their main data. The reason is that many researchers use the pre-analysed data contained within these proprietary files and will likely not decide not to reformat their data if they would loose this information.

Reply:
EDF stands for two separate things:

- european data format (EEG and so on), open, community-developed format
- eyelink data format (eyetracking data), proprietary format